tasks/sep/base-005-fp-holocene-upgrade/README.md

```diff
@@ -0,0 +1,35 @@
+# Holocene Hardfork Upgrade - Base Sepolia
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the Fault Proof contracts of Base Sepolia for the Holocene hardfork.
+
+This upgrades the Fault Proof contracts in the
+[op-contracts/v1.8.0-rc.1](https://github.com/ethereum-optimism/optimism/tree/op-contracts/v1.8.0-rc.1) release.
+
+## Pre-deployments
+
+- `MIPS` - `0x69470D6970Cd2A006b84B1d4d70179c892cFCE01`
+- `FaultDisputeGame` - `0xB7fB44a61fdE2b9DB28a84366e168b14D1a1b103`
+- `PermissionedDisputeGame` - `0x68f600e592799c16D1b096616eDbf1681FB9c0De`
```

Comment:
These look like legitimate new deployments for Base Sepolia (`l2ChainId` is `84532`)